docker/project-fortis-interfaces/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:6.12.0
+
+ADD . /app
+WORKDIR /app
```

Comment:
I'm wondering if instead of doing the `ADD`+`WORKDIR` thing, we do something with `volumes:` like this in the docker-compose.yml file instead:

```
    ports:
        - 8888:80
    volumes:
      - ./:/app
```

This way, any changes done from the container can be used or committed to git. What do you think?